amazon/dynamodb/client/src/main/kotlin/org/http4k/connect/amazon/dynamodb/mapper/DynamoDbIndexMapper.kt

```diff
@@ -18,7 +18,7 @@ class DynamoDbIndexMapper<Document : Any, HashKey : Any, SortKey : Any>(
     private val dynamoDb: DynamoDb,
     private val tableName: TableName,
     private val itemLens: BiDiLens<Item, Document>,
-    private val schema: DynamoDbTableMapperSchema<HashKey, SortKey>
+    internal val schema: DynamoDbTableMapperSchema<HashKey, SortKey>
```

Comment:
I wonder if we should move the extension `query`, `scan`, and `count` functions from `DynamoDbQueryDsl.kt` into the `DynamoDbIndexMapper`, then this change wouldn't be necessary. And doing it now would be a good time, since it changes the imports on the user's side.

Reply:
Done